justfile

```diff
@@ -480,7 +456,7 @@ action-lint:
 
 # Ensure all devcontainer versions are in sync
 action-dev-check:
-    bin/action-dev-check
+    action-dev-check
```

Comment:
Do we need this recipe anymore? I don't think this is doing anything.

Reply:
```
:; rg action-dev-check .github 
.github/workflows/actions.yml
27:      - run: just action-dev-check
```

Reply:
it should probably be part of `lint`, too

Reply:
I saw it's being used, but is the recipe doing anything? Isn't this a recipe calling itself?

```shell
$ just action-dev-check
action-dev-check
```

Reply:
Oh okay this is calling `linkerd/dev`'s `action-dev-check` recipe.